presto-docs/src/main/sphinx/develop/connector.rst

```diff
@@ -0,0 +1,62 @@
+==========
+Connectors
+==========
+
+First, if you have not already looked at :doc:`/develop/spi-overview`, check there for
```

Comment:
I added it at the request of @KBP-TDC because he was confused about how to write a function based on the docs (I think he just jumped to the function section). And, for example, the function development section in the Postgres docs is self-contained and you can jump right to it. I don't feel too strongly about it, though, so I can remove the pointers to the overview if you'd like.